### DISCUSSION TITLE
[2023.1.29] FTB模组包｜FTB Mods.zip

### DISCUSSION THREAD

**Discussion by IcedDog** [DM-Earth/Cabricality#47] [General]
<sup>最后更新｜Last Updated - `ftb-library<~>build.167` `ftb-quests<~>build.157` `ftb-teams<~>build.88` `quests-additions<~>1.4.1`</sup>
[`下载｜Download`](https://github.com/DM-Earth/Cabricality/files/10530017/FTB.Mods.zip)

**Comment by JieningYu** [comment]
小心ARR协议

**Comment by Hypersale** [comment]
nice,that's just what i need
